pkg/api/bytes.go

```diff
@@ -88,3 +97,38 @@ func (s *server) bytesGetHandler(w http.ResponseWriter, r *http.Request) {
 		s.Logger.Errorf("bytes download: data read %s", address)
 	}
 }
+
+func (s *server) createTag(w http.ResponseWriter, r *http.Request) *tags.Tag {
+	// if tag header is not there create a new one
+	var tag *tags.Tag
+	tagUidStr := r.Header.Get(TagHeaderUid)
+	if tagUidStr == "" {
+		tagName := fmt.Sprintf("unnamed_tag_%d", time.Now().Unix())
+		var err error
+		tag, err = s.Tags.Create(tagName, 0, false)
+		if err != nil {
+			s.Logger.Debugf("bytes upload: tag creation error: %v", err)
```

Comment:
is this common practice in bee to have debug and error lines? why not just error the first line

Reply:
yes it is part of coding protocol the @janos presented starting out.